src/eureka/S4_generate_lightcurves/s4_genLC.py

```diff
@@ -222,6 +222,12 @@ def genlc(eventlabel, ecf_path=None, s3_meta=None, input_meta=None):
                 meta.wave_low = meta.wave-neg_dwav
                 meta.wave_hi = meta.wave+pos_dwav
                 meta.nspecchan = len(meta.wave)
+            elif hasattr(meta, 'wave_input') and meta.wave_input is not None:
+                # bins defined by file input. 2 columns: low and high edges
+                meta.wave_low = np.genfromtxt(meta.wave_input, usecols=[0])
+                meta.wave_hi = np.genfromtxt(meta.wave_input, usecols=[1])
```

Comment:
This can more simply be doing using:
```
meta.wave_low, meta.wave_hi = np.genfromtxt(meta.wave_input).T
```

Note that I'm not 100% sure that you'll need to use the `.T` to transpose the array, but I think you will